packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -218,6 +218,7 @@
     }
     &__symbol {
         margin-left: 0.4rem;
+        font-weight: var(--text-weight-bold);
```

Comment:
we are remove all css which is related to text and create text component so I suggest you instead of add css use `weight` 
<Text size='xxxs' className='dc-contract-card__symbol' weight="bold">